actiontext/lib/generators/action_text/install/install_generator.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+require "pathname"
+require "json"
+
+module ActionText
+  module Generators
+    class InstallGenerator < ::Rails::Generators::Base
+      source_root File.expand_path("templates", __dir__)
+
+      def install_javascript_dependencies
+        template "#{__dir__}/../../../../../railties/lib/rails/generators/rails/app/templates/bin/yarn.tt",
+                 "bin/yarn"
+
+        say "Installing JavaScript dependencies"
+        run "yarn add #{js_dependencies.map { |name, version| "#{name}@#{version}" }.join(" ")}",
+          abort_on_failure: true, capture: true
+      end
+
+      def append_dependencies_to_package_file
+        app_javascript_pack_path = Pathname.new("app/javascript/packs/application.js")
+
+        if app_javascript_pack_path.exist?
+          js_dependencies.keys.each do |name|
+            line = %[require("#{name}")]
+
+            unless app_javascript_pack_path.read.include? line
+              say "Adding #{name} to #{app_javascript_pack_path}"
+              append_to_file app_javascript_pack_path, "\n#{line}"
+            end
+          end
+        else
+          warn <<~WARNING
+            WARNING: Action Text can't locate your JavaScript bundle to add its package dependencies.
+
+            Add these lines to any bundles:
+
+            require("trix")
+            require("@rails/actiontext")
+
+            Alternatively, install and setup the webpacker gem then rerun `bin/rails action_text:install`
+            to have these dependencies added automatically.
+          WARNING
+        end
+      end
+
+      def create_actiontext_files
+        template "actiontext.scss", "app/assets/stylesheets/actiontext.scss"
+
+        copy_file "#{__dir__}/../../../../app/views/active_storage/blobs/_blob.html.erb",
+                  "app/views/active_storage/blobs/_blob.html.erb"
+      end
+
+      def create_migrations
+        run "rake active_storage:install:migrations"
+        run "rake railties:install:migrations"
+        run "rake action_text:install:migrations"
```

Comment:
Should/can we assume that `action_text:install:migrations` has set up its dependencies properly and only run it?

Reply:
Which dependencies are you concerned about? I'd say if there are dependencies, it should check and warn the user if something is missing.

Reply:
Sorry, I meant task dependencies. If `action_text:install:migrations` require `railties:install:migrations` and `active_storage:install:migrations` first, then shouldn't the Action Text task have those tasks as dependencies?

Reply:
I have addressed two of your comments. However, I apologize, but it's still not clear to me what code changes implicate from having tasks as dependencies. I didn't find examples.

Reply:
Let me rephrase just for the record. I would expect the Rake set up to be `task "action_text:install:migrations": [ "active_storage:install:migrations", "railties:install:migrations" ]`. In Rake's parlance when I run the Action Text migration task first run the Active Storage and Railties' migration tasks.

Anyway, I checked up on it and that's not how the tasks are defined. So this is fine.